src/environments/tests.py

```diff
@@ -34,4 +36,59 @@ def test_create_identity_should_assign_relevant_attributes(self):
         self.assertIsInstance(identity.environment, Environment)
         self.assertTrue(hasattr(identity, 'created_date'))
 
+    def test_get_all_feature_states(self):
+        feature = Feature.objects.create(name="Test Feature", project=self.project)
+        feature_2 = Feature.objects.create(name="Test Feature 2", project=self.project)
+        environment_2 = Environment.objects.create(name="Test Environment 2", project=self.project)
+
+        identity_1 = Identity.objects.create(
+            identifier="test-identity-1",
+            environment=self.environment,
+        )
+        identity_2 = Identity.objects.create(
+            identifier="test-identity-2",
+            environment=self.environment,
+        )
+        identity_3 = Identity.objects.create(
+            identifier="test-identity-3",
+            environment=environment_2,
+        )
+
+        # User unassigned - automatically should be created via `Feature` save method.
```

Comment:
The two unassigned calls to get FeatureState.objects in this section seem unnecessary?

Reply:
@matthewelwell thanks for the review; I will get to this as soon as I can (probably in the next few days)

Reply:
Thanks @sjdines. Let me know if there's anything I can do to help.